functions.php

```diff
@@ -52,7 +52,7 @@ function twentyten_posted_on() {
 						} else if ( $i < count( $external_authors ) - 1 ) {
 							$author_html .= ' ' . __( ' and ' );
 						}
-						$i++;
+						$i ++;
```

Comment:
I am not a fan of this style. In fact I have never seen it been used in practice before. Is there any reasoning behind it?